drivers/tee/optee/optee_smc.h

```diff
@@ -215,6 +215,9 @@ struct optee_smc_get_shm_config_result {
  */
 #define OPTEE_SMC_SEC_CAP_DYNAMIC_SHM		BIT(2)
 
+/* Secure world supports Shared Memory with a NULL buffer reference */
+#define OPTEE_SMC_SEC_CAP_MEMREF_NULL		BIT(4)
```

Comment:
BTW why not BIT(3)?

Reply:
> BTW why not BIT(3)?

We have OPTEE_SMC_SEC_CAP_VIRTUALIZATION on that bit.

Reply:
https://github.com/OP-TEE/optee_os/blob/master/core/arch/arm/include/sm/optee_smc.h#L277-L278

I.e., it's not in sync with that one.

Reply:
Ah, OK thanks.